x-pack/plugins/lens/public/editor_frame_service/editor_frame/editor_frame.scss

```diff
@@ -1,5 +1,4 @@
 @import 'data_panel_wrapper';
-@import 'expression_renderer';
 @import 'frame_layout';
 @import 'suggestion_panel';
 @import 'workspace_panel_wrapper';
```

Comment:
Sorry one more thing. Only `_index.scss` files should be manifest files. You'll need to either revert this back to an index file or put all the imports of these files in their respective JS files.